tutoraspects_unidigital/patches/superset-extra-assets

```diff
@@ -0,0 +1,5 @@
+{% include 'openedx-assets/Aggregation_Dashboard_43.yaml' %}
```

Comment:
Does the _43, _44, _45 suffix mean something? if I don't have access to the previous 42 does it make a difference if we remove them?

I think we could benefit from a prefix like campus_ or unidigital_ better than the number.

Reply:
No, we can remove the prefix, it was just to keep the standard.

Is there really a benefit for that? I can rename the folder from `openedx-assets` to `openedx-assets-unidigital`